src/lib/helpers/liquid.ts

```diff
@@ -17,7 +17,6 @@ const disabled = {
   },
 } as any;
 
-engine.registerTag("render", disabled);
```

Comment:
We'll need to enable this eventually since 'render' tags will be allowed. I think it's fine to just ship this now, but happy to hold off and do another PR/CLI release when the project is closer to shipping.

Reply:
Sounds good, I'll remove that change. 

We support the exact same syntax as what liquidjs does. The only difference is that we expect a partial key instead of a file name, but they're still both just strings.

Reply:
> The only difference is that we expect a partial key instead of a file name, but they're still both just strings.

It sounds some differences - like normally `.liquid` extension unless [`extname` option is set](https://liquidjs.com/api/interfaces/LiquidOptions.html#extname)? And you can use `with` or `for` tags in it?